src/Services/RemoteAppOptions.cs

```diff
@@ -45,4 +45,9 @@ public class RemoteAppOptions
     /// </summary>
     public HttpClient? BackchannelHttpClient { get; set; }
 #endif
+
+    // To ensure secure API keys are used, enforce that
+    // keys are parsable as GUIDs
+    public static bool Validate(RemoteAppOptions options) =>
```

Comment:
Does this need to be public? Would internal work?

Reply:
Good catch. I'll make it internal.